client/sites/components/sites-dashboard.tsx

```diff
@@ -347,25 +347,27 @@ const SitesDashboard = ( {
 		}
 	};
 
-	const openSitePreviewPane = (
-		site: SiteExcerptData,
-		source: 'site_field' | 'action' | 'list_row_click' | 'environment_switcher',
-		openInNewTab?: boolean
-	) => {
-		recordTracksEvent( 'calypso_sites_dashboard_open_site_preview_pane', {
-			site_id: site.ID,
-			source,
-		} );
-		showSitesPage(
-			`/${ FEATURE_TO_ROUTE_MAP[ initialSiteFeature ].replace( ':site', site.slug ) }`,
-			openInNewTab
-		);
+	const sitePreviewPane = {
+		getUrl: ( site: SiteExcerptData ) => {
+			return `/${ FEATURE_TO_ROUTE_MAP[ initialSiteFeature ].replace( ':site', site.slug ) }`;
+		},
```

Comment:
When you observe the current behavior, you'll notice that changing the Dataview settings (like filters or sorting) will add query params to the current URL, and clicking a site from the list will navigate to the site preview pane URL with the same query params appended.

That's fine, except I have doubts about whether this behavior is intuitive when the user command-clicks the link to open in a new tab. It feels weird that opening a site preview pane "permalink" in a new tab carries over transient app state that is irrelevant to the permalinked object. The "Close" button is also kind of unnatural in this context. Just me? 🤔 

Anyway, at least for the actual `href` that the anchor element receives in markup, I think it should just be the clean URL and not appended with query params.

Reply:
I noticed A4A doesn't bother to maintain the dataview filter state via url, I think dropping this behavior will be ok.